pkg/gateway/model_build_targetgroup_test.go

```diff
@@ -277,9 +276,8 @@ func Test_TGModelByHTTPRouteBuild(t *testing.T) {
 			name: "Create LatticeService where backend K8S service does NOT exist",
 			httpRoute: &gateway_api.HTTPRoute{
 				ObjectMeta: metav1.ObjectMeta{
```

Comment:
When test httproute creation (wantIsDeleted is false),  HTTPRoute's DeletionTimestamp should be 0